docs/docs/install/configuration.md

```diff
@@ -0,0 +1,34 @@
+| DJANGO_DATABASE_KEY    | ***Internal*** The key used for referencing the [Database config information](https://docs.djangoproject.com/en/4.2/ref/settings/#databases). There should be atleast one `default` database config.                                                                                                                                                                                                                                                                                                                                                | `default`                    |
+| DJANGO_DATABASE_URL    | A Postgres connection string of the database which will be used for storing metadata related to Mathesar in the format of: `postgres://{user}:{password}@{hostname}:{port}/{database-name}`. For example, for a database role whose username is `mathesar` and  `mathesar` is the password, `mathesar_db` is the database name and `5432` is the port under which the postgres server is listening on and `mathesar_django` is the database name, then the connection string will be`postgres://mathesar:mathesar@mathesar_db:5432/mathesar_django` | *Required                    |
+| MATHESAR_DATABASES     | A dictionary of database connection string of the external databases to be managed by Mathesar keyed by a unique database key. It should of the format `(<unique_db_key>\|postgres://{user}:{password}@{hostname}:{port}/{database1-name}),(<unique_db_key>\|postgres://{user}:{password}@{hostname}:{port}/{database2-name}),...`.                                                                                                                                                                                                                 | *Required                    |
+| DJANGO_SETTINGS_MODULE | ***Internal*** Relative path to the django settings file                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | `config.settings.production` |
+| MODE                   | ***Internal*** The mode under which the Frontend files will be served. The values can be either be `DEVELOPMENT` or `PRODUCTION`. Running under `DEVELOPMENT` mode will run a Svelte server and enable hot reloading of the files, while running under `PRODUCTION` mode will serve the static assets from the `/static/` url                                                                                                                                                                                                                       | `PRODUCTION`                 |
```

Comment:
The idea is to let users know that these environment variables should not be modified. Ideally, we should disable it in the production image but it is still there in case we need it for debugging production servers